README.md

```diff
@@ -20,10 +20,7 @@ To work on this extensions, you first need to install VS Code and nodejs. Afterw
   ```bash
   yarn
   ```
-- Install a VS Code extension necessary for development:
-  ```bash
-  code --install-extension amodio.tsl-problem-matcher
-  ```
+- Install the extensions VS Code recommends
```

Comment:
I like doing this on the command line, since it is also used in the previous steps. Maybe we can just say something like "Install the recommended extensions in VS Code, or run the following command line: "?

Reply:
That would work, but then we'd have to update this README every time we have some new extension required (like the discussions about enforcing code formatting. I have a suggestion that might help both of us. Let me push into this PR and you can see if you are OK with it.